Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+NativeTranslation.swift

```diff
@@ -18,6 +18,7 @@ extension JNISwift2JavaGenerator {
   
   struct NativeJavaTranslation {
     let javaPackage: String
+    let javaClassLookupTable: [String: String]
```

Comment:
Can you please document here what the value and what the key are expected to be? Can be docs comment or typealias